client/src/components/Reviews/ReviewList/ReviewList.jsx

```diff
@@ -2,29 +2,86 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import ReviewListEntry from './ReviewListEntry/ReviewListEntry.jsx';
 
+const SORT_BY = ['Relevant', 'Helpful', 'Newest'];
+
 class ReviewList extends React.Component {
-  constructor (props) {
+  constructor(props) {
     super(props);
-    this.state = {};
+    this.state = {
+      listLength: 2,
+      sortBy: SORT_BY[0]
+    };
+    this.incrementListLength = this.incrementListLength.bind(this);
+    this.handleSelectorChange = this.handleSelectorChange.bind(this);
+  }
+
+  incrementListLength() {
+    let { listLength } = this.state;
+    listLength += 2;
+    this.setState({ listLength });
+    // TODO: make GET request for next page of reviews
+    // if listLength >= .75 reviews.length
   }
 
-  render () {
-    let { reviews } = this.props;
+  handleSelectorChange(e) {
+    let idx = e.target.value;
```

Comment:
I suggest giving a more descriptive name. Although it's clear that you're getting the SORT_BY index after reading the code, it would make it more readable if it was something more descriptive, like sortingOption or something like this. (I'm not very good at naming variables, so maybe my example it's not that descriptive either.)